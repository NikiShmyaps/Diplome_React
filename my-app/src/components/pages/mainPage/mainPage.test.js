import React from 'react';
import MainPage from './mainPage';
import {shallow} from 'enzyme';

describe('Testing <MainPage/>', ()=> {
  it('MainPage have rendered correctly', ()=>{
    const mainPage = shallow(<MainPage/>)
    expect(mainPage).toMatchSnapshot();
  })
}); 