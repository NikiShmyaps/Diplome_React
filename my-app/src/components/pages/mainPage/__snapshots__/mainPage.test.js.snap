// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <MainPage/> MainPage have rendered correctly 1`] = `ShallowWrapper {}`;